examples/model_selection/plot_roc.py

```diff
@@ -135,23 +252,23 @@
 plt.plot(
     fpr["macro"],
     tpr["macro"],
-    label="macro-average ROC curve (area = {0:0.2f})".format(roc_auc["macro"]),
+    label=f"macro-average ROC curve (AUC = {roc_auc['macro']:.2f})",
     color="navy",
     linestyle=":",
     linewidth=4,
 )
 
 colors = cycle(["aqua", "darkorange", "cornflowerblue"])
```

Comment:
Colorblind compatibility was first introduced in [this commit](https://github.com/scikit-learn/scikit-learn/commit/83430819de36c99e5a06c20cff90bcb880df6715). Even though some things have changed since then, I verified that the plots are still colorblind friendly :)